### ISSUE TITLE
Problem running code on Mac 

### ISSUE THREAD

**Issue by r63b** [graysuit/facebrute#2]
I get the following error when I try to run the code in Mac. I have installed all of the packages needed. Most of the errors sound like they are coming from sockets but I don't know enough about sockets or about python to try and fix it.

`Jakes-Air:facebrute-master jc$ python3 fb.py
May 01 16:01:48.184 [notice] Tor 0.4.2.7 running on Darwin with Libevent 2.1.11-stable, OpenSSL 1.1.1g, Zlib 1.2.11, Liblzma N/A, and Libzstd N/A.
May 01 16:01:48.185 [notice] Tor can't help you if you use it wrong! Learn how to be safe at https://www.torproject.org/download/download#warning
May 01 16:01:48.185 [warn] Command-line option 'start' with no value. Failing.
May 01 16:01:48.185 [err] Reading config failed--see warnings above.

 Coded by : http://github.com/graysuit
 Meet him : http://fb.com/gray.programmerz.5 
 Wordlist Generator : http://github.com/graysuit/wGen
 [+] Enter Username : example
 [+] Enter Wordlist name, Default:password.txt : 
 [+] Should I display Public IP? Default:y, y/n : n
 [+] Removing duplicate words in 
 [+] Only passwords >= 6 will be used.
 [+] Open exampleresponse.txt to view complete responses by facebook.

Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/stem/socket.py", line 535, in _make_socket
    control_socket.connect((self.address, self.port))
ConnectionRefusedError: [Errno 61] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "fb.py", line 62, in <module>
    if function(email,passw):break
  File "fb.py", line 36, in function
    with Controller.from_port(port = 9051) as c:
  File "/usr/local/lib/python3.7/site-packages/stem/control.py", line 1033, in from_port
    control_port = stem.socket.ControlPort(address, port)
  File "/usr/local/lib/python3.7/site-packages/stem/socket.py", line 503, in __init__
    self.connect()
  File "/usr/local/lib/python3.7/site-packages/stem/socket.py", line 172, in connect
    self._socket = self._make_socket()
  File "/usr/local/lib/python3.7/site-packages/stem/socket.py", line 538, in _make_socket
    raise stem.SocketError(exc)
stem.SocketError: [Errno 61] Connection refused
Jakes-Air:facebrute-master jc$ `

![Screenshot 2020-05-01 at 16 02 50](https://user-images.githubusercontent.com/4274904/80815495-aae25800-8bc5-11ea-839f-a0e46ffa7810.png)

**Comment by graysuit** [comment]
Before putting efforts you should read this before:
https://github.com/graysuit/facebrute/blob/master/README.md#-no-more-working-2019-march-30

Does tor is started ?
Write this command to check tor status:
`sudo service tor status`

**Comment by r63b** [comment]
The command doesn't work and gives an error "**sudo: service: command not found**". But when I type brew services start tor I get the code **"Successfully started `tor` (label: homebrew.mxcl.tor)**" so tor seems to be starting fine.

**Comment by graysuit** [comment]
Edit fb.py,
Change port from 9051 To 9050
https://github.com/graysuit/facebrute/blob/master/fb.py#L36

See whether it works.

**Comment by graysuit** [comment]
Sorry I forget that you are using MAC,
So there will be alternative for


*sudo may be brew*



*Your error has been reported 3 times,May be its some where fault in my
script,But script is not good anymore,Because tor proxies are now falling
down.*

**Comment by graysuit** [comment]
Sorry I forget that you are using MAC,
So there will be alternative for sudo may be brew*



*Your error has been reported 3 times,May be its some where fault in my
script,But script is not good anymore,Because tor proxies are now falling
down.*
…

**Comment by r63b** [comment]
I've tried changing the os.system code so it works with the mac commands and tried different IPs and ports for the proxies. I've checked all the packages are installed correctly and have checked that I don't have a firewall blocking the port. I've also tried running as sudo and even rebooted my Mac to see if any of the packages needed a system reboot to work. 

I'm going to give up because as you said the code is not good anymore. Thank you for trying to help me :)
